static/django_templates/base/index.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles %}
+{% load staticfiles compress %}
```

Comment:
reason for disabling compressing is because django-compressor/master does not support django 1.9 yet, dev branch is fixed now